metrics/replicacpumem.go

```diff
@@ -0,0 +1,84 @@
+package metrics
```

Comment:
Since this file covers both `replica` and `client` CPU and memory metrics, I suggest renaming the file to `cpumem.go` instead.